polkadot/xcm/pallet-xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -157,15 +155,29 @@ benchmarks_instance_pallet! {
 		);
 		let sender_account_balance_before = T::TransactAsset::balance(&sender_account);
 
+		// generate holding and add possible required fees
```

Comment:
Where do you add the possible required fee?

Reply:
right bellow - `required fees = expected_assets_in_holding`
```
            for a in expected_assets_in_holding.into_inner() {
				holding.push(a);
			}
```

Reply:
there was just `executor.set_holding(expected_assets_in_holding`) before, which could possibly remove previously `executor.set_holding(holding.clone()`